lib/player.js

```diff
@@ -487,6 +492,19 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     /** @private {shaka.util.CmcdManager} */
     this.cmcdManager_ = null;
 
+    // This is the canvas elemnent that will be used for rendering LCEVC
+    // enhannced frames.
+    /** @private {?HTMLCanvasElement} */
+    this.canvas_ = null;
+
+    // This is the LCEVC Dil object to decode LCEVC.
+    /** @private {?shaka.lcevc.Dil} */
+    this.lcevcDil_ = null;
+
+    // This is the LCEVC DIl configuration object.
+    /** @private {?Object} */
```

Comment:
This needs a better type.

Reply:
This is changed to `LcevcDil.LcevcDilConfig` type for clarity.